packages/manager/apps/dedicated/client/app/dedicated/server/details/server.controller.js

```diff
@@ -352,6 +355,10 @@ export default class ServerCtrl {
       this.launchBringYourOwnImagePolling();
     }
 
+    const cpannelEolDate = new Date('2023-07-31');
```

Comment:
**nitpick:** Use `feature-flipping` to manage dates / enable / disable banner and so on.